### PR TITLE
filter subdepartments

### PR DESCRIPTION
**Pull request by loldot**
- [x] New feature
- [ ] Bug fix
- [ ] High impact

**Description of work:**

**Testing:**
- [ ] Can be tested
- [ ] Automatic tests created / updated
- [x] Local tests are passing

**Checklist:**
- [x] Considered automated tests
- [x] Considered updating specification / documentation
- [x] Considered work items 
- [x] Considered security
- [x] Performed developer testing
- [ ] Checklist finalized / ready for review

### CODE REVIEW

File: src/backend/api/Fusion.Resources.Domain/Queries/GetDepartmentPersonnel.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public GetDepartmentPersonnel(string department, ODataQueryParams? queryParams =
             QueryParams = queryParams;
         }
 
-
+        public bool IncludeSubdepartments { get; set; } = false;
```
Comment (HansDahle) [inline src/backend/api/Fusion.Resources.Domain/Queries/GetDepartmentPersonnel.cs:26]: ..SubDepartment?

File: src/backend/api/Fusion.Resources.Api/Controllers/Personnel/InternalPersonnelController.cs [hunk 2]
```diff
@@ -83,9 +94,11 @@ public InternalPersonnelController()
 
             #endregion
 
+            var command = new GetDepartmentPersonnel(fullDepartmentString, query)
+                .WithTimeline(shouldExpandTimeline, timelineStart, timelineEnd);
+            command.IncludeSubdepartments = includeSubdepartments;
```
Comment (HansDahle) [inline src/backend/api/Fusion.Resources.Api/Controllers/Personnel/InternalPersonnelController.cs:99]: Should follow the same pattern with fluent. 
`.IncludeSubDepartments(includeSubDepartments)`

**Review by HansDahle — APPROVED** [review]
Nice job, just some minor non-critical issues that could be improved now, or added to list of TODOs.. up to you ;)

File: src/backend/api/Fusion.Resources.Domain/Queries/GetDepartmentPersonnel.cs (hunk 1 above)
Comment (loldot) [reply src/backend/api/Fusion.Resources.Domain/Queries/GetDepartmentPersonnel.cs:26]: According to Merriam Webster subdepartment is the most common spelling. https://www.merriam-webster.com/dictionary/subdepartment

**Pull request merged**